gravitee-apim-e2e/api-test/applications/application-membership.spec.ts

```diff
@@ -53,7 +51,7 @@ describe('Applications - Membership', () => {
     group = await configurationManagementApiAsAdminUser.createGroup({
       orgId,
       envId,
-      newGroupEntity: GroupsFaker.newGroup({ event_rules: [{ event: 'API_CREATE' }] }),
+      newGroupEntity: GroupsFaker.newGroup({ event_rules: [{ event: 'APPLICATION_CREATE' }] }),
```

Comment:
It's probably ok like this or we use the fixed value from the defined enum. 
```suggestion
      newGroupEntity: GroupsFaker.newGroup({ event_rules: [{ event: GroupEvent.APPLICATION_CREATE}] }),
```

Reply:
@ctschacher `GroupEvent.` is not official generated type.